README.md

```diff
@@ -12,10 +12,10 @@ languages in [SonarQube](http://www.sonarqube.org/), [SonarCloud](https://sonarc
 
 ## Features
 
-* 380+ C# rules and 130+ VB.&#8203;NET rules
-* Metrics (complexity, duplications, number of lines etc.)
-* Import of [unit test results](https://docs.sonarqube.org/x/CoBh) from VSTest, NUnit and xUnit

```

Comment:
[we don't push the results to SQ](https://github.com/SonarSource/sonar-dotnet/issues/886#issuecomment-596393473)

Reply:
actually we do 🤦‍♂️ @costin-zaharia-sonarsource , see [here](https://discuss.sonarsource.com/t/net-drop-importing-unit-test-results-from-our-products-documentation-and-guides/7516/6?u=andrei_epure). Sorry for this. I will send later a PR ... but anyway the link was broken.

I will also have to give an update on #886 to correct myself.